instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './CommentSection.css'
+import PropTypes from 'prop-types';
+import CommentForm from '../CommentForm';
+// import Comment from '../c';
+
+// import PostContainer from './PostContainer/PostContainer';
+
+class CommentSection extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            comments: props.comments
+        }
+    }
+
+addNewComment = (event, indexNum, commentText) => {
+    console.log(this.state.comments)
+    const copiedComments = this.state.comments
+    const arrayWithNewComment = copiedComments.push(commentText)
```

Comment:
Note that the return value of .push() is not what you'd expect. .push() actually modifies the array you call it on